### PR TITLE
Add Grid Element

### PR DESCRIPTION
**Pull request by fcescob**
This PR introduces a Responsive Grid Element

### CODE REVIEW

File: atlas/packages/app/src/shared/components/Grid/Grid.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  grid-template-columns: repeat(${(props) => `auto-${props.repeat}`}, minmax(min(270px, 100%), 1fr));
+`
+
+type GridProps = {
+  gap?: number | string
+  repeat?: 'fit' | 'fill'
+  className?: string
+  onResize?: (sizes: number[]) => void
+}
+const Grid: React.FC<GridProps> = ({ children, className, repeat = 'fit', onResize, ...props }) => {
```
Comment (kdembler) [inline atlas/packages/app/src/shared/components/Grid/Grid.tsx]: Shouldn't `gap` have some default value here or in the `Container`?

File: atlas/packages/app/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx [hunk 2]
```diff
@@ -26,8 +26,7 @@ const InfiniteVideoGrid: React.FC<InfiniteVideoGridProps> = ({
   initialLoading,
   className,
 }) => {
-  // TODO: base this on the container width and some responsive items/row
-  const videosPerRow = VIDEOS_PER_ROW
+  const [videosPerRow, setVideosPerRow] = useState(VIDEOS_PER_ROW)
```
Comment (kdembler) [inline atlas/packages/app/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx]: Now I think it'd make sense to rename the constant to `INITIAL_VIDEOS_PER_ROW` or something alike

File: atlas/yarn.lock [hunk 3]
```diff
@@ -16994,7 +16983,7 @@ redeyed@~2.1.0:
   dependencies:
     esprima "~4.0.0"
 
-redux@*, redux@^4.0.0, redux@^4.0.5:
+redux@*, redux@^4.0.0:
```
Comment (kdembler) [inline atlas/yarn.lock]: There's still `redux` left here? Does some other dependency rely on it?

File: atlas/packages/app/src/shared/components/Grid/Grid.tsx (hunk 1 above)
Comment (fcescob) [reply atlas/packages/app/src/shared/components/Grid/Grid.tsx]: Good idea, i'll add `spacing.xl` as a default value

File: atlas/packages/app/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx (hunk 2 above)
Comment (fcescob) [reply atlas/packages/app/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx]: Good catch, will change it

File: atlas/yarn.lock (hunk 3 above)
Comment (fcescob) [reply atlas/yarn.lock]: There were `@types/redux` and `@types/react-redux` left in our dependencies, removed both

**Review by kdembler — APPROVED** [review]

**Review by bedeho — APPROVED** [review]

**Pull request merged**